### DISCUSSION TITLE
Page URLs have .html suffix

### DISCUSSION THREAD

**Discussion by DougInAMug** [honkit/honkit#406] [Q&A]
Although the Honkit example site manages to avoid this issue without naming every page `README.md` in it's own nested folder, all the other example pages seem to follow this approach. Those which deviate from this pattern end up with the suffix, e.g. https://www.algorithm-archive.org/contents/barnsley/barnsley.html

Does anyone have an idea how to achieve suffix-free page URLs without the `<FolderName>/README.md` pattern?

**Comment by azu** [comment]
> `<FolderName>/README.md` pattern?

I've used this pattern in https://github.com/asciidwango/js-primer/tree/master/source
website: https://jsprimer.net/intro/

**Reply by DougInAMug** [reply]
So you are saying it _is_ necessary to use the `<FolderName>/README.md` pattern to avoid `.html` suffixes?
